src/uct/ib/base/ib_md.c

```diff
@@ -158,7 +158,7 @@ ucs_config_field_t uct_ib_md_config_table[] = {
      "Maximal number of invalidated memory keys that are kept idle before reuse.",
      ucs_offsetof(uct_ib_md_config_t, ext.max_idle_rkey_count), UCS_CONFIG_TYPE_UINT},
 
-    {"REG_RETRY_CNT", "7",
+    {"REG_RETRY_CNT", "0",
```

Comment:
maybe use "inf"?

Reply:
@Artemy-Mellanox
wdyt about making it `inf` for gtest only? I'd not set inf by default for all cases

Reply:
assumption is that this behavior is not specific to gtest, a regular application will experience the same crashes and therefore we make the `inf` default

Reply:
do not we expect it to succeed with the reasonable number of attempts? Imo it is better to crash after X attempts rather than hang

Reply:
Imo this would be justified if we encountered a hang situation or at least expected it

Reply:
hang is quite generic syndrome, may be not easy to find a correlation with this code when it occurs.
My point is to set it to some relatively large value, like `128` or `256`, but ok, it's up to you